moto/glue/urls.py

```diff
@@ -2,6 +2,6 @@
 
 from .responses import GlueResponse
 
-url_bases = ["https?://glue(.*).amazonaws.com"]
+url_bases = [r"https?://glue\.(.*)\.amazonaws\.com"]
```

Comment:
`url_bases = [r"https?://glue\.(.+)\.amazonaws\.com"]`

If region is mandatory for Glue, it should be a plus not a star.